src/partials/options.html

```diff
@@ -1,327 +1,317 @@
+            ng-model="ctrl.panel.prefix"
             ng-change="ctrl.render()"
             ng-model-onblur
           />
         </div>
       </div>
-    </div>
-  </div>
-
-  <!--
-    Based ob singlestat editor plugin
-    https://github.com/grafana/grafana/blob/v4.1.1/public/app/plugins/panel/singlestat/editor.html
-  -->
-  <!-- TODO: support colors customizing for multibar mode -->
-  <div class="section gf-form-group" ng-if="!ctrl.isMultibar">
-    <h5 class="section-heading">Coloring<h5>
-    <div class="gf-form">
-      <label class="gf-form-label width-8">Type</label>
-      <div class="gf-form-select-wrapper width-11">
-        <select
-          class="gf-form-input"
-          ng-model="ctrl.panel.coloringType"
-          ng-options="n for n in ctrl.coloringTypeOptions"
+      <div class="gf-form">
+        <label class="gf-form-label width-6">Postfix</label>
+        <input
+          type="text" class="gf-form-input width-10"
+          ng-model="ctrl.panel.postfix"
           ng-change="ctrl.render()"
+          ng-model-onblur
+        />
+      </div>
+      <div class="gf-form">
+        <label class="gf-form-label width-6">Decimals</label>
+        <input
+          type="number"
+          class="gf-form-input width-10"
+          placeholder="auto" data-placement="right"
+          bs-tooltip="'Override automatic decimal precision'"
+          ng-model="ctrl.panel.decimals"
+          ng-change="ctrl.render()"
+          ng-model-onblur
         />
       </div>
-    </div>
 
-    <div class="gf-form">
-      <label class="gf-form-label width-8">Opacity</label>
-      <input
-        type="number"
-        min="0"
-        max="1"
-        step="0.1"
-        class="gf-form-input width-11"
-        ng-model="ctrl.panel.opacity"
-        ng-blur="ctrl.render()"
-      />
+      <br>
+      <div class="section gf-form-group">
+        <h5 class="section-heading">Key Labels</h5>
+        <div class="gf-form">
+          <label class="gf-form-label width-6">Title Type</label>
+          <div class="gf-form-select-wrapper width-12">
+            <select
+              class="gf-form-input"
+              ng-model="ctrl.panel.titleViewType"
+              ng-options="n for n in ctrl.titleViewTypeOptions"
+              ng-change="ctrl.render()"
+            />
+          </div>
+        </div>
+        <div class="gf-form-inline">
+          <div class="gf-form">
+            <label class="gf-form-label width-6">
+              Alias
+              <tip>
+                You can insert HTML here.
+                Use $__key variable to access the original key
+              </tip>
+            </label>
+            <!-- TODO: better style -->
+            <textarea
+              class="gf-form-input"
+              ng-model="ctrl.panel.alias"
+              ng-change="ctrl.render()"
+              ng-model-onblur
+            />
+          </div>
+        </div>
+      </div>
     </div>
+фвыфывфывфвы
```

Comment:
Do we need it?)

Reply:
It was not the task, but I agree with you and will fix it here

Reply:
I've made it like this:

![image](https://user-images.githubusercontent.com/1022757/99419358-518d0c80-28fc-11eb-8754-aa804fdbd283.png)